app/controllers/dashboard_controller.rb

```diff
@@ -2,7 +2,7 @@ class DashboardController < ApplicationController
   def index
     @recent_activities = policy_scope(PublicActivity::Activity)
                          .limit(20)
-                         .order("created_at desc")
+                         .order(id: :desc)
```

Comment:
Why have you changed the ordering (here and in other places) ? Does it make such a difference ?

Reply:
In one word: performance. Since the activities are supposed to be immutable and we won't have a new one being added in the middle of the others, I decided to go with `id` since it's the primary key and it's faster to order compared to a date column that doesn't have an index on it. I don't remember how better it was, but it was better.

I think we still need to keep #1278 open because my improvements were good compared to what we had but still not good enough IMHO.